donkeycar/parts/mavlink.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+from __future__ import print_function
+"""
+Script for using a Pixhawk running any ArduPilot- or PX4-based code with Donkeycar 
+
+author: @zlite (Chris Anderson), 2022
+contrib: @wallarug (Cian Byrne) 2019
+contrib: @peterpanstechland 2019
+contrib: @sctse999 2020
+
+If you're getting a permissions error on the USB port: "sudo chmod 666 /dev/ttyACM0"
```

Comment:
Instead advice users to add some group membership and remove modem manager.
Not sure if DFU rules are needed. Just took an example of STM32 hardware so you can change accordingly.

On Ubuntu:
```
$ sudo usermod -a -G dialout $USER
$ sudo usermod -a -G plugdev $USER
$ sudo apt-get remove modemmanager
$ (echo '# DFU (Internal bootloader for STM32 MCUs)'
 echo 'ACTION=="add", SUBSYSTEM=="usb", ATTRS{idVendor}=="0483", ATTRS{idProduct}=="df11", MODE="0664", GROUP="plugdev"') | sudo tee /etc/udev/rules.d/45-stdfu-permissions.rules > /dev/null
```